runtime/src/test/java/org/eclipse/xpanse/runtime/modules/common/OpenApiGeneratorJarManageTest.java

```diff
@@ -43,6 +44,7 @@ void testGetOpenApiWorkdir() {
     }
 
     @Test
+    @Disabled
```

Comment:
@baixinsui  we must enable this test again.

Reply:
Enabled